theme/static/js/hs_file_browser.js

```diff
@@ -612,7 +612,7 @@ function bindFileBrowserItemEvents() {
 
             var fileAggType = [];
             // main-file is available on the aggregation folder and only single file aggregations have a data-pk of 1 on the file
-            if ($(event.target).closest("li").attr("main-file") || $(event.target).closest("li").attr("data-pk") === "1"){
+            if ($(event.target).closest("li").attr("main-file") || $(event.target).closest("li").attr("single-file") === "true"){
```

Comment:
@sblack-usu is case sensitivity a consideration factor here? I am just thinking ```is_single_file_aggregation``` property on a logical file is a Boolean, which when converted to string in templates, could be ```True```?

Reply:
done